src/test/java/org/apache/accumulo/testing/continous/BulkKey.java

```diff
@@ -0,0 +1,236 @@
+package org.apache.accumulo.testing.continous;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+import org.apache.accumulo.core.data.ByteSequence;
+import org.apache.accumulo.core.data.Key;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.io.WritableComparable;
+import org.apache.hadoop.io.WritableComparator;
+import org.apache.hadoop.io.WritableUtils;
+
+/**
+ * Supports immediate sorting via eager deser of the key object. This has the benefit of reducing
+ * the amount of deserialization that may occur when sorting keys in memory
+ */
+public class BulkKey implements WritableComparable<BulkKey> {
```

Comment:
yes it was only provided as comparison. I don't intend this to be in the final instance. I was just showing you some examples.

Reply:
It's used in the unit tests to provide a point of comparison and reference -- but was planning on removing it since you are the only one with any reference to what it is.